tests/dataset_processor_test.py

```diff
@@ -233,7 +235,7 @@ def test_dataset_processor_with_numerical_column():
                         "<task 1>convert to text2text\nExample:\nfoo\nLabel:\n",
                         "<task 1>convert to text2text\nExample:\nbar\nLabel:\n",
                     ],
-                    "model_output": ["foo", "bar", "0", "1"],
+                    "model_output": ["baz", "qux", "0", "1"],
```

Comment:
Why is the output changing here? Was the test broken before?

Reply:
Yes, I believe it was broken but the assert was not catching it properly. I haven't looked deeply into this though.